integration_tests/pages/page.ts

```diff
@@ -138,6 +142,21 @@ export default abstract class Page {
     })
   }
 
+  shouldContainRadioButtons(options: Array<{ label: string; value: string }>): void {
```

Comment:
Are the contents of the array here `GovukFrontendRadiosItem`s (possibly `WithLabel` to assert that they aren't `undefined`)? If that's the right type, this might be more of a `shouldContainRadioItems` method with an `items` argument